kratos/python_scripts/auxiliar_methods_adaptative_remeshing.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics
+
+def AdaptativeRemeshingRunSolutionLoop(analysis):
```

Comment:
it looks better now but still it is quite messy IMO

not blocking, but we should carefully evaluate this, otherwise in the end I am afraid none will understand any more what is going on

Reply:
OK